tools/autograd/gen_variable_type.py

```diff
@@ -277,7 +277,7 @@ def format_trace_inputs(declaration):
         trace_name = uninplace_api_name(declaration['api_name'])
         has_factory_name = trace_name in FACTORY_FUNCTION_NAMES
         if has_factory_name:
-            outplace = ADD_TRACE_INPUT.substitute(name='result', input='result.options()')
+            outplace = ADD_TRACE_INPUT.substitute(name='out', input='out.options()')
```

Comment:
shouldn't this already have a name from declarations.yaml?

Reply:
This only does a substitution. Using a specific name and passing that through will follow later in conjunction with [this PR](https://github.com/pytorch/pytorch/pull/16253).